### PR TITLE
chore: add types to exported plugins

### LINKED ISSUE #23: fix (create-dm-app): add type for exported list of plugins (see ../demo-app/index.tsx)

### PR DESCRIPTION
**Pull request by KristianKjerstad**
closes #23

### CODE REVIEW

**Comment by soofstad** [comment]
I think the idea with #23 was to add these types to dm-core, so it can be imported in this app

File: src/plugins/demo-app/index.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { EDmtPluginType } from '@development-framework/dm-core'
 
 import App from './App'
+import {TPlugin} from "./Types";
```
Comment (netr0m) [inline src/plugins/demo-app/index.tsx:4]:
```suggestion
import { TPlugin } from "./Types";
```
(PS: Do you have pre-commit hooks installed? I believe this should be auto-fixed by prettier (?))

File: src/plugins/demo-app/Types.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
-export {
+import {EDmtPluginType} from "@development-framework/dm-core";
```
Comment (netr0m) [inline src/plugins/demo-app/Types.ts:1]:
```suggestion
import { EDmtPluginType } from "@development-framework/dm-core";
```

File: src/plugins/demo-app/Types.ts [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+import React from "react";
```
Comment (netr0m) [inline src/plugins/demo-app/Types.ts:2]:
```suggestion
import React from 'react'
```

**Review by netr0m — APPROVED** [review]
Couple of remarks on syntax. Looks good otherwise!

**Comment by KristianKjerstad** [comment]
> I think the idea with #23 was to add these types to dm-core, so it can be imported in this app

that makes sense.

Will create a new PR on dm-core!

**Comment by KristianKjerstad** [comment]
> Couple of remarks on syntax. Looks good otherwise!

nice, seems to be an issue with eslint in my setup

**Review by netr0m — APPROVED** [review]

**Pull request merged**